src/node.cc

```diff
@@ -704,6 +704,12 @@ const char *signo_string(int signo) {
 # endif
 #endif
 
+#ifdef SIGINFO
+# if SIGINFO != SIGPWR
```

Comment:
I'm happy to defer to you on this, but I do want to clarify that I gave thought to this initially and determined that Node is _not_ being built with `-Wundef`, which would print the nasty warnings you mentioned.

(In fact, V8 relies heavily upon undefined symbols, and adding `-Wundef` to the `cflags` defined within `common.gypi` creates thousands of warnings throughout both the V8 and the Node source.)

I'll update the branch momentarily.